torchtune/modules/kv_cache.py

```diff
@@ -40,7 +40,6 @@ def __init__(
         self.register_buffer(
             "v_cache", torch.zeros(cache_shape, dtype=dtype), persistent=False
         )
-        self.size = 0
```

Comment:
If you remove this variable, you need to update MultiHeadAttention too because it uses self.kvcach.size